WordPress/wporg-showcase-2022
# Add related sites to single site pages

Fixes #140 — This adds the related sites section to single site pages. It uses Jetpack's related posts feature, but overrides the output to render using the theme's blocks. It also adds some fallback posts to make sure there are always 3 results in the related section.

| Design | Branch |
|---|---|
| <img width="1350" alt="" src="https://github.com/WordPress/wporg-showcase-2022/assets/541093/dfbecbdb-09e5-4958-8b73-3e3ba967ee4f"> | ![](https://github.com/WordPress/wporg-showcase-2022/assets/541093/e27151e8-48c1-4641-8994-65b6de1919fb) |
| <img width="416" alt="" src="https://github.com/WordPress/wporg-showcase-2022/assets/541093/454c694d-e4d2-4d6b-aa5f-ad30a8ad67cf"> | ![](https://github.com/WordPress/wporg-showcase-2022/assets/541093/40290dce-3f02-4a3c-80fc-e199e7e8c452) |

@WordPress/meta-design The heading font size (for "Related sites") uses the "heading 5" font size, which is 26px on large screens, but breaks to 20px on small. In figma, it's 26 across both sizes, is that intentional? Should I set this to a custom 26px instead?

**To test:**

This needs to be tested on a live site, like the staging site, so use your sandbox. This is because Jetpack's related posts does not work in dev/offline mode.

- View a single site
- It should have 3 posts below the pagination
- Sites without related content will use the latest 3 featured sites
- Older sites will have real related content, for example https://wordpress.org/showcase-v2/the-city-university-of-new-york/
- Sites with less than 3 related posts will fill in with featured sites
